e2e/codebuild/run-e2e.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+eval "$(pyenv init -)"
+cd ${PR_REPO_NAME}
+cd e2e/tests/
+
+FAILED=0
+
+function failed() {
+    FAILED=1
+    echo "TEST FAILED: ${t} python version: ${ver}"
+}
+
+for ver in "$@" ; do
+    pyenv global ${ver}
+    for t in $(ls -1 *.sh) ; do
```

Comment:
This looks ripe for unintended results. Are you *sure* you want to do this?

Reply:
elaborate?

Reply:
Nah, fam